src/base-element.js

```diff
@@ -24,6 +24,8 @@ import {isArray, toWin} from './types';
 import {isExperimentOn} from './experiments';
 import {preconnectForElement} from './preconnect';
 
+const TAG_ = 'Base-element';
```

Comment:
This isn't useful debugging data since every element will have the same tag.

Reply:
I thought the message itself was useful enough and would provide adequate info and thought the tag name was just useful indicating where the error was originating in terms of which class. Changed the error to use element's tagName which will be AMP component name.